### PR TITLE
Researcher role + dashboard

### PR DESCRIPTION
**Pull request by OptimusRhine**
https://www.pivotaltracker.com/story/show/150112113

* added new Researcher role
* added new Research Dashboard ('/research')
  * only available to those with Researcher role
  * includes header/gil links to new page, for those with the role
* moved 'Export Questionnaire Data' functionality to new /research page
* moved data export population logic into admin.js

### CODE REVIEW

File: portal/static/js/admin.js [hunk 1]
```diff
@@ -359,5 +334,67 @@ AdminTool.prototype.getInstrumentList = function() {
...
 
+AdminTool.prototype.handleDownloadModal = function() {
+
+    var self = this;
+     /* 
+      *populate instruments list based on user's parent org
+      */
+    $("#dataDownloadModal").on('shown.bs.modal', function () {
+          var parentOrgList = AT.getUserTopLevelParentOrgs(AT.getUserOrgs());
+          console.dir(parentOrgList)
```
Comment (OptimusRhine) [inline portal/static/js/admin.js]: @achen2401 ^ I think that's a question for you

**Comment by OptimusRhine** [comment]
@pbugni good point - added in that redirect from /home for researchers (happens AFTER the potential redirect check for staff/intervention_staff)

File: portal/static/js/admin.js (hunk 1 above)
Comment (pbugni) [inline portal/static/js/admin.js]: I'm referring to the `console.dir()` call - I assume that was a debugging statement left behind?  IIRC some <cough> browsers don't do well with lots of console logging.

**Comment by pbugni** [comment]
@OptimusRhine github shows a merge conflict -- would you mind resolving?

**Comment by OptimusRhine** [comment]
@pbugni should be resolved now

**Pull request merged**